jest-puppeteer-react.config.js

```diff
@@ -0,0 +1,76 @@
+const webpack = require("webpack")
+const HtmlWebpackPlugin = require("html-webpack-plugin")
+const path = require("path")
+const debug = require("debug")("jest-puppeteer-react")
+const isCI = typeof process.env.CI !== "undefined"
+const isMac = process.platform === "darwin"
+
+function getIPAddress() {
+  const interfaces = require("os").networkInterfaces()
+  for (let devName in interfaces) {
+    const iface = interfaces[devName]
+
+    for (let i = 0; i < iface.length; i++) {
+      const alias = iface[i]
+      if (
+        alias.family === "IPv4" &&
+        alias.address !== "127.0.0.1" &&
+        !alias.internal
+      )
+        return alias.address
+    }
+  }
+
+  return "0.0.0.0"
+}
```

Comment:
This is necessary for running against localhost?

Reply:
No idea, came with the template ( it runs inside a docker container ) so probably is